src/frontend/Felles/PersonHeader/Visittkort.tsx

```diff
@@ -162,7 +168,7 @@ const VisittkortComponent: FC<{ data: IPersonopplysninger; behandling?: Behandli
                             gåTilUrl(`/person/${fagsakPersonId}`);
                         }}
                     >
-                        <Visningsnavn>{navn.visningsnavn}</Visningsnavn>
+                        <Visningsnavn>{`${utledVisningsnavn()}`}</Visningsnavn>
```

Comment:
```suggestion
                        <Visningsnavn>{utledVisningsnavn()}</Visningsnavn>
```

Tror dette burde være nok